test/object-store/baas/app_mongo.cpp

```diff
@@ -0,0 +1,703 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#if REALM_ENABLE_SYNC
+#if REALM_ENABLE_AUTH_TESTS
+
+#include <util/collection_fixtures.hpp>
+#include <util/baas_admin_api.hpp>
+
+#include <realm/object-store/sync/mongo_client.hpp>
+#include <realm/object-store/sync/mongo_database.hpp>
+#include <realm/object-store/sync/mongo_collection.hpp>
+#include <realm/object-store/util/bson/bson.hpp>
+
+using namespace realm;
+using namespace realm::app;
+using util::Optional;
+
+
+// MARK: - Remote Mongo Client Tests
+
+TEST_CASE("app: remote mongo client", "[sync][app][baas][mongo][new]") {
```

Comment:
what is the purpose of the `new` tag?

Reply:
This was to indicate that these tests were moved to the new _baas/_ directory and will eventually be moved to a new executable (and simplified testing of the new tests during development). I was planning to remove this tag once all the tests have been separated/moved.

Reply:
gotcha 👍🏼